src/torchmetrics/functional/classification/precision_recall_curve.py

```diff
@@ -622,7 +632,7 @@ def _multilabel_precision_recall_curve_update(
     len_t = len(thresholds)
     # num_samples x num_labels x num_thresholds
     preds_t = (preds.unsqueeze(-1) >= thresholds.unsqueeze(0).unsqueeze(0)).long()
-    unique_mapping = preds_t + 2 * target.unsqueeze(-1)
+    unique_mapping = preds_t + 2 * target.unsqueeze(-1)  # target.long().unsqueeze(-1)
```

Comment:
Is the comment intented to be there?

Reply:
right, forgot to remove :)
Thanks
```suggestion
    unique_mapping = preds_t + 2 * target.unsqueeze(-1)
```